Wireframe/index.html

```diff
@@ -7,27 +7,49 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
-    <main>
-      <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+     <header>
+  <h1>
+    THIS IS A TITLE
+  </h1>
+  <p>
+    This is a template webpage for the first sprint of the onboarding module backlog.<br> It contains three articles exploring fundemental development concepts: the purpose of README files in the documentation, why are wireframes useful in the design process, and how git branches function in version control systems.<br>All of which are considered fundemental knowledge for web developement and collaborative project workflows.
+  </p>
+</header>
+<main>
+  <article id="first_article_wrap">  
+    <img src="https://git-scm.com/book/en/v2/images/basic-merging-1.png">
+    <h3>Branches in Git</h3>
+    <address>
+      Written by a shared effort,
+      <time datetime="2025-05-09">on 2025/05/09</time>
```

Comment:
A time of writing doesn't feel like it belongs in an `<address>` tag - can you think of a more suitable tag to use for this information?

(This applies to all three articles)